data/sea/54-zebra.h

```diff
@@ -155,7 +156,7 @@ static int64_t zebra_translate_column
                 istring_t *strings = anemone_mempool_alloc (mempool, count * sizeof (istring_t));
                 int offset = 1;
                 for (int i = 0; i != count; ++i) {
-                    int64_t index = i + elem_start;
+                    int64_t index = i;
```

Comment:
example probably better

```
nested array of strings
  ["beef", "helps", "keep", "order"]

lengths:
  [4, 5, 4, 5]
scan (data.a.n):
  [0, 4, 9, 13, 18]
values (data.a.table):
  "beefhelpskeeporder"

zebra_translate_table(elem_start = 0, count = 1, table)
= ["beef"]

zebra_translate_table(elem_start = 0, count = 2, table)
= ["beef", "helps"]

zebra_translate_table(elem_start = 2, count = 2, table)
= ["keep", "order"]
```

the last one is the most interesting because it needs to start looking at scan array 2 and 3 (`data.a.n`), so to pull out the value "beef" it looks up `scan[2] = 9` for the start index, and `scan[3] = 13` for the end index, meaning the length is `13-9=4`.